var/spack/repos/builtin/packages/ferret/package.py

```diff
@@ -15,13 +15,15 @@ class Ferret(Package):
     homepage = "http://ferret.pmel.noaa.gov/Ferret/home"
     url      = "ftp://ftp.pmel.noaa.gov/ferret/pub/source/fer_source.v696.tar.gz"
 
+    version('7.2', '21c339b1bafa6939fc869428d906451f130f7e77e828c532ab9488d51cf43095')
     version('6.96', '51722027c864369f41bab5751dfff8cc')
 
-    depends_on("hdf5~mpi~fortran")
-    depends_on("netcdf~mpi")
+    depends_on("hdf5+hl")
+    depends_on("netcdf")
```

Comment:
THANK YOU for making this change.  Using `~` in `depends_on()` is a bad idea (and unnecessary) because it can cause things to not concretize.  They're little time bombs that crop up repeatedly and make things not work:

https://github.com/spack/spack/issues/6231
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/spack/0Lq90S3Yojc/ZykeNEd8BQAJ